README.md

```diff
@@ -1,2 +1,15 @@
 # shed-store
- A one stop for all your purchase
+
+### About
+SHED-STORE is  one stop platform for all your fashion purchase.
+
+### Preview
```

Comment:
be little more description about your Site add pages link or images

Look into - https://readme.so/ 
for reference